src/vmm/src/vmm_config/machine_config.rs

```diff
@@ -178,6 +201,9 @@ pub struct MachineConfigUpdate {
     /// The memory size in MiB.
     #[serde(default)]
     pub mem_size_mib: Option<usize>,
+    /// The memory configuration
+    #[serde(default)]
+    pub mem_config: Option<MemoryConfig>,
```

Comment:
is it possible for us to update it post boot?

Reply:
no